website_sale_product_detail_attribute_image/__manifest__.py

```diff
@@ -19,7 +19,4 @@
         'views/product_attribute_views.xml',
         'views/templates.xml',
     ],
-    'demo': [
-        'data/demo.xml',
```

Comment:
The file still exists. Either restore it here in the manifest, or remove the `demo.xml` file please.